PowerPointLabs/PowerPointLabs/ActionFramework/EffectsLab/ColorizeBackground/ColorizeBackgroundImageHandler.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Drawing;
+using PowerPointLabs.ActionFramework.Common.Attribute;
+using PowerPointLabs.ActionFramework.Common.Interface;
+
+namespace PowerPointLabs.ActionFramework.EffectsLab
+{
+    [ExportImageRibbonId(TextCollection.ColorizeBackgroundMenuId)]
+    class ColorizeBackgroundImageHandler : ImageHandler
+    {
+        protected override Bitmap GetImage(string ribbonId)
+        {
+            return new Bitmap(Properties.Resources.RecolorRemainder);
```

Comment:
These methods are "Colorize...", should it be made consistent to "Recolor" since it is called that in PPTLabs?